test/ResourceFactoryTest.php

```diff
@@ -15,20 +15,23 @@
 
 class ResourceFactoryTest extends TestCase
 {
-    protected $testWriter = null;
-    protected $resourceFactory = null;
+    /** @var TestAsset\ConfigWriter */
+    protected $testWriter;
+
+    /** @var ResourceFactory */
+    protected $resourceFactory;
 
     protected function setUp(): void
     {
         $this->resourceFactory = new ResourceFactory(
-            $this->getMockBuilder(ModuleUtils::class, [], [], '', false)
+            $this->getMockBuilder(ModuleUtils::class)
```

Comment:
AFAIK that's the default behaviour of `$this->createMock()`

Reply:
Recent versions (and the ones allowed with current commits on this branch) only have a single argument; Psalm had flagged that for me, which is why I made the change.

Reply:
Ah, missed that you were talking `createMock()`, vs `getMockBuilder()`; updated!